packages/turf-bezier/index.js

```diff
@@ -33,9 +33,9 @@ var Spline = require('./spline.js');
  * };
  *
  * var curved = turf.bezier(line);
+ * curved.properties = { stroke: '#0f0' };
  *
  * //addToMap
- * curved.properties = { stroke: '#0f0' };
  * var addToMap = [line, curved]
  */
```

Comment:
so rather than moving the `curved.properties` above the `//addToMap` tag, I generally move it to below the `var addToMap = [line, curved]`. That benefit of that is that it isn't rendered as part of the code sample (which might potentially confuse people)

So I'd make it look like
````
  * var curved = turf.bezier(line);
   *
   * //addToMap
   * var addToMap = [line, curved];
   * curved.properties = { stroke: '#0f0' };
   */
````